src/core/public/core_system.ts

```diff
@@ -123,6 +129,7 @@ export class CoreSystem {
     this.legacyPlatform.stop();
     this.notifications.stop();
     this.loadingCount.stop();
+    this.chrome.stop();
```

Comment:
😬 This is why i've been switching to a `stop$` observable rather than manually calling stop in so many places, because I keep forgetting about things that need to be stopped...